client/src/components/Leaflet.js

```diff
@@ -68,6 +86,13 @@ const icon = new Icon({
   shadowSize: [41, 41]
 })
 
+const icon2 = new Icon({
+  iconUrl: MARKER_FLAG_BLUE,
```

Comment:
Do we also need a retina icon here as in https://github.com/NCI-Agency/anet/blob/4850220e6d3ef626c834e546eb85f20ef0069366/client/src/components/Leaflet.js#L80 ?

Reply:
I am assuming, marker icons are for engagements, marker flags for locations, that is why we don't include here?

Reply:
one line below` iconRetinaUrl: MARKER_FLAG_BLUE_2X,`